### ISSUE TITLE
Configure extensions to apply formatting to

### ISSUE THREAD

**Issue by karlgrz** [mauve/vscode-terraform#41]
Great plugin, thanks!

It would be great to allow for other extensions. For example, we use jinja2 templates to generate our terraform .tf files, so in source control our file extensions are .tf.tpl, which this plugin ignores. 

It would be nice to be able to add that extension in a configuration setting, or if the plugin applied this formatting to any file extension containing .tf in it. 

:man_shrugging: 

Thanks again!

**Comment by jlucktay** [comment]
Can't you configure this file association already, with base VSCode functionality, with something like this in settings?

```json
    "files.associations": {
        "*.tf.tpl": "terraform"
    },
```

**Comment by mauve** [comment]
Sadly the plugin uses this check to check whether to apply formatting:

https://github.com/mauve/vscode-terraform/blob/master/src/helpers.ts#L3

So it *right now* only allows `.tf` and `.tfvars` files.

Wonder if it makes sense to just keep the `languageId` check and skip the extension check or if this should be made configurable, any ideas?

**Comment by ChristianBoehlke** [comment]
The plugin has been working very well for us!

Our [Nomad](https://www.nomadproject.io) configuration is also written in HCL and we would like to be able to configure `*.nomad` files to be included as well.

**Comment by jlucktay** [comment]
For my 2p, the `languageId` check is the bare minimum for now.

**Comment by mauve** [comment]
Hi, this should now be fixed in version 0.0.20, please give it a try! Thanks for your patience.
